lib/ext/tags.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
```

Comment:
I see. I was expecting that ES5 will only be in the packaged version uploaded to npm. But checking it sounds good to, as long as the generator is stable and won't create spurious differences between commits.